stable/meteor/templates/cronjob.yaml

```diff
@@ -33,6 +33,20 @@ spec:
               - -c
               - |
                 meteor run /opt/recipes;
+                METEOR_EXIT_CODE=$?
+                echo "Meteor command exited with code METEOR_EXIT_CODE"
```

Comment:
do we intend to produce the actual code value here?

Reply:
hmm, actually not sure too haha, wdyt mas? I simply thought it's easier to see why the auto clean up not running if it happened. But I just remember we also can see more detail about it in meteor logs hehe

Reply:
I changed my mind, it is no need to echo there hehe thank you mas